yarn-project/noir-contracts/src/contracts/schnorr_multi_key_account_contract/src/main.nr

```diff
@@ -0,0 +1,93 @@
+mod storage;
+mod public_key_note;
+
+// Account contract that uses Schnorr signatures for authentication.
+// The signing key is stored in an immutable private note and should be different from the signing key.
```

Comment:
```suggestion
// The signing key is stored in an immutable private note and should be different from the encryption key.
```

Not sure if we really call it encryption key so please correct me here if it's not the case.

Reply:
Hah, good catch!